tiledb/sm/query/reader.cc

```diff
@@ -737,6 +737,16 @@ Status Reader::set_offsets_extra_element(bool add_extra_element) {
   return Status::Ok();
 }
 
+int Reader::offsets_bitsize() const {
+  return offsets_bitsize_;
+}
+
+Status Reader::set_offsets_bitsize(int bitsize) {
```

Comment:
- `bitsize` can be `const uin32_t` here
- Let's check that `bitsize` is either 32 or 64, erroring otherwise.